dags/main_summary.py

```diff
@@ -43,6 +45,79 @@
     email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
     dag=dag)
 
+main_ping_bigquery_export = gke_command(
+    task_id="main_ping_bigquery_extract",
+    command=[
+        "bq",
+        "extract",
+        "--destination_format=AVRO",
+        "moz-fx-data-shared-prod.payload_bytes_decoded.telemetry_telemetry__main_v4${{ds_no_dash}}",
+        "gs://moz-fx-data-derived-datasets-parquet-tmp/export/submission_date={{ds}}/document_namespace=telemetry/document_type=main/document_version=4/*.avro"
+    ],
+    docker_image="google/cloud-sdk:263.0.0-slim",
+)
+
+main_summary_dataproc = SubDagOperator(
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name="main_summary",
+        dag_name="main_summary_dataproc",
+        default_args=default_args,
+        cluster_name="main-summary-{{ds}}",
+        image_version="1.3",
+        idle_delete_ttl="600",
+        worker_machine_type="n1-standard-8",
+        num_preemptible_workers=40,
+        optional_components=[],
+        install_component_gateway=False,
+        jar_urls=[
+            "s3a://net-mozaws-data-us-west-2-ops-ci-artifacts/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar"
+        ],
+        main_class="com.mozilla.telemetry.views.MainSummaryView",
+        jar_args=[
+            "--from={{ds_nodash}}",
+            "--to={{ds_nodash}}"
+            "--bucket=gs://moz-fx-data-derived-datasets-parquet-tmp",
```

Comment:
@jasonthomas is there a different bucket I should be using here instead of `gs://moz-fx-data-derived-datasets-parquet-tmp` that won't interference with the production import of main summary in `gs://moz-fx-data-derived-datasets-parquet`?